std_msgs/String name
vigir_generic_params/ParameterMsg[] params
